consensus/src/consensus.rs

```diff
@@ -307,11 +313,7 @@ impl<R: ConsensusRpc> ConsensusClient<R> {
             .map_err(|_| eyre!("could not fetch bootstrap"))?;
 
         if bootstrap.header.slot > base_slot {
-            if let Some(stored_tip_slot) = self.storage().get_tip_beacon_header_slot()? {
-                if bootstrap.header.slot > stored_tip_slot {
-                    return Err(ConsensusError::CheckpointTooNew.into());
-                }
-            }
```

Comment:
When verifier reaches slot `M+N` and its `initial_checkpoint` is set to slot `M`, we restart it.
At starting, the `bootstrap.header.slot` will be smaller than or equal to `stored_tip_slot`, it should be acceptable: the verifier will not missing any blocks.

Reply:
ok, in this case, we can just ensure the condition of `bootstrap.header.slot < stored_tip_slot`.